recipes/abacus/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "abacus" %}
+{% set version = "3.2.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmodeling/abacus-develop/archive/v{{ version }}.tar.gz
+  sha256: 65a84a4a02a085c42f86d54b487bf3f1501a45f940370a8a7527c881145f78d6
+
+build:
+  script: |
+    export CMAKE_PREFIX_PATH=`python -c "import torch;print(torch.__path__[0])"`/share/cmake:$CMAKE_PREFIX_PATH
+    cmake -B conda_build ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release -DENABLE_DEEPKS=1 -DENABLE_LIBXC=1
+    cmake --build conda_build -j`nproc`
+    cmake --install conda_build
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+
+  host:
+    - {{ mpi }}
+    - openblas
+    - elpa * mpi_{{ mpi }}_*
+    - fftw * mpi_{{ mpi }}_*
+    - cereal
+    - pytorch
+    - libxc
+
+  run:
+    - {{ mpi }}
+    - scalapack
+    - libopenblas
+    - openblas
+    - libxc-c
```

Comment:
There is an overdepending warning for libxc. We don't allow static linking.

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=752531&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=4117

Reply:
I'll chime in only because I was the one who split up the libxc and libxc-c packages. There are no static libs in either (https://github.com/conda-forge/libxc-feedstock/blob/main/recipe/meta.yaml#L72). Use the `libxc` package if you need the Fortran interface and `libxc-c` if you only need C interface. Both have `run_exports`, so you can put it in `host` with no need for it in `run`.

Reply:
I see. This package does not contain Fortran, so I think libxc should be replaced by libxc-c.